lib/linguist/generated.rb

```diff
@@ -103,6 +104,13 @@ def cocoapods?
       !!name.match(/(^Pods|\/Pods)\//)
     end
 
+    # Internal: Is the blob part of Carthage/Build/, which contains dependencies not meant for humans in pull requests.
+    #
+    # Returns true or false.
+    def carthage_build?
+      !!name.match(/(^Carthage|\/Carthage)\/Build\//)
```

Comment:
Could we use the shorter syntax `(^|\/)Carthage` here?

Reply:
I copied CocoaPods' without thinking, but you're right!

Reply:
Whoopsie 🙈

Reply:
Changed, tests still pass.